diff --git a/node_modules/@expo/metro-runtime/src/messageSocket.native.ts b/node_modules/@expo/metro-runtime/src/messageSocket.native.ts
index 1d7c933..fba9ada 100644
--- a/node_modules/@expo/metro-runtime/src/messageSocket.native.ts
+++ b/node_modules/@expo/metro-runtime/src/messageSocket.native.ts
@@ -1,7 +1,8 @@
 /* eslint-env browser */
 
 function createWebSocketConnection(path: string = '/message') {
-  const getDevServer = require('react-native/Libraries/Core/Devtools/getDevServer');
+  const getDevServerMod = require('react-native/Libraries/Core/Devtools/getDevServer');
+  const getDevServer = getDevServerMod.default ?? getDevServerMod;
   const devServer = getDevServer();
   if (!devServer.bundleLoadedFromServer) {
     throw new Error('Cannot create devtools websocket connections in embedded environments.');
@@ -9,7 +10,8 @@ function createWebSocketConnection(path: string = '/message') {
 
   const devServerUrl = new URL(devServer.url);
   const serverScheme = devServerUrl.protocol === 'https:' ? 'wss' : 'ws';
-  const WebSocket = require('react-native/Libraries/WebSocket/WebSocket');
+  const WebSocketMod = require('react-native/Libraries/WebSocket/WebSocket');
+  const WebSocket = WebSocketMod.default ?? WebSocketMod;
   return new WebSocket(`${serverScheme}://${devServerUrl.host}${path}`);
 }
 
