diff --git a/node_modules/expo-router/build/getDevServer/index.native.js b/node_modules/expo-router/build/getDevServer/index.native.js
index f17cd60..286ba67 100644
--- a/node_modules/expo-router/build/getDevServer/index.native.js
+++ b/node_modules/expo-router/build/getDevServer/index.native.js
@@ -1,5 +1,6 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.getDevServer = void 0;
-exports.getDevServer = require('react-native/Libraries/Core/Devtools/getDevServer');
+const _getDevServer = require('react-native/Libraries/Core/Devtools/getDevServer');
+exports.getDevServer = _getDevServer.default ?? _getDevServer;
 //# sourceMappingURL=index.native.js.map
\ No newline at end of file
